### DISCUSSION TITLE
Mapped or virtual static folders

### DISCUSSION THREAD

**Discussion by GitMcClintock** [nuxt/nuxt#17210] [Questions]
In my Nuxt (nuxt3 rc6) app I would like to be able to set up a virtual folder of static content (images).  In my ASP.NET days I would map a virtual folder in IIS, which makes any folder in the accessible file system appear as a subfolder within the website or app.  I am working in Windows 10, so I could possibly use a Windows shortcut to achieve this, but it would be more transparent to configure this withing Nuxt if that is possible.  I would like to be able to map any folder on the system to appear as a subfolder to say /assets or /public, so the client has direct access to that folder of images.

It is not practical or desirable to copy the folder into my Nuxt project, so I'd like to map to it.

I am new to Nuxt, but have Vue, node, and express experience.

**Comment by simchor** [comment]
Any luck on that problem? Did you end up finding a solution? I am actually looking for a answer for that. I also comes from ASP NET
